app/org/sagebionetworks/bridge/dynamodb/DynamoUploadFieldDefinition.java

```diff
@@ -45,6 +46,27 @@ public boolean isRequired() {
         return type;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        DynamoUploadFieldDefinition that = (DynamoUploadFieldDefinition) o;
+        return Objects.equal(required, that.required) &&
+                Objects.equal(name, that.name) &&
+                Objects.equal(type, that.type);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public int hashCode() {
+        return Objects.hashCode(name, required, type);
```

Comment:
Neither did I. Apparently, IntelliJ had a setting for generating equals() and hashCode() using Guava, so I used it, and this is what happened.